cfgov/v1/__init__.py

```diff
@@ -56,6 +56,9 @@ def parse_links(soup):
     icon_pattern = re.compile(settings.EXTERNAL_ICON_PATTERN)
     a_class = os.environ.get('EXTERNAL_A_CSS', 'icon-link icon-link__external-link')
     span_class = os.environ.get('EXTERNAL_SPAN_CSS', 'icon-link_text')
+    for tag in soup:
+        if hasattr(tag, 'style'):
+            del tag['style']
```

Comment:
This affects any sheer page content coming from elasticsearch and any content wrapped in the `process_links` context processor. In the hero's case, that's [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/molecules/hero.html#L57).